drivers/gpu/drm/vc4/vc4_crtc.c

```diff
@@ -210,6 +210,9 @@ static u32 vc4_get_fifo_full_level(struct vc4_crtc *vc4_crtc, u32 format)
 {
 	const struct vc4_crtc_data *crtc_data = vc4_crtc_to_vc4_crtc_data(vc4_crtc);
 	const struct vc4_pv_data *pv_data = vc4_crtc_to_vc4_pv_data(vc4_crtc);
+	struct drm_crtc *crtc = &vc4_crtc->base;
+	struct drm_device *dev = crtc->dev;
+	struct vc4_dev *vc4 = to_vc4_dev(dev);
```

Comment:
Slight nitpicking but you don't need crtc, so you may as well do dev = vc4_crtc->base.dev (if I've got my indirections right). Or as dev isn't actually needed either, 
```
struct vc4_dev *vc4 = to_vc4_dev(vc4_crtc->base.dev);
```